### DISCUSSION TITLE
KeyError: "['FOD_ID', 'DISCOVERY_DATE', 'FIRE_SIZE'] not in index"

### DISCUSSION THREAD

**Discussion by annieM459** [earthlab-education/Earth-Analytics-2023-01-Intro#492] [Q&A06 Vector Data]
Hi! For the step in he wildfire notebook where we create the fire_gdf, it doesn't bring in FIRE_SIZE, 'FOD_ID', 'DISCOVERY_DATE', and insetad looks like this: 
<img width="1434" alt="Screenshot 2023-10-11 at 4 09 30 PM" src="https://github.com/earthlab-education/Earth-Analytics-2023-01-Intro/assets/140002702/d454b76d-c88a-43ec-a5d3-70373013b345">

Which causes problems later. Any suggestions?

**Comment by annettebalmes** [comment]
Hi Annie,

I think your problem lies outside of this cell. I believe we both have the same code for that cell:

```python
import pyogrio

fire_path = os.path.join(fire_dir, 'Data', 'FPA_FOD_20221014.gdb')

if not 'fire_gdf_raw' in globals():

    print("fire_gdf_raw does not exist. Loading...")

    fire_gdf_raw = pyogrio.read_dataframe(
        fire_path, layer='Fires'
    )

fire_gdf_raw.head()
```

And this is part of my output which does contain the right columns:

![image](https://github.com/earthlab-education/Earth-Analytics-2023-01-Intro/assets/110803009/cd83acd9-62d1-4e65-a7db-7976aae5e467)

Here are some of my thoughts:

1.  Double check you have the right url in the cell above and call the right url in your et.data
```python
fire_url = (
    "https://www.fs.usda.gov/rds/archive/products/RDS-2013-0009.6/"
    "RDS-2013-0009.6_Data_Format2_GDB.zip"
)

fire_dir = et.data.get_data(url=fire_url)
fire_dir
```

2. Make sure you are in the right directory at the time you are running the code. The following is mine for example:
```python
'/home/jovyan/earth-analytics/data/earthpy-downloads/RDS-2013-0009.6_Data_Format2_GDB'
```

3. If you changed your directory or updated your url after you ran your script the first time to create fire_gdf, it will not update because fire_gdf will already exist and no new data will pull. Hopefully restarting your codespace would fix that.

**Comment by annieM459** [comment]
You're right! Got it! Thank you.
